server/schedule/checker/replica_strategy.go

```diff
@@ -70,33 +70,37 @@ func (s *ReplicaStrategy) SelectStoreToAdd(coLocationStores []*core.StoreInfo, e
 
 	isolationComparer := filter.IsolationComparer(s.locationLabels, coLocationStores)
 	strictStateFilter := &filter.StoreStateFilter{ActionScope: s.checkerName, MoveRegion: true}
-	target := filter.NewCandidates(s.cluster.GetStores()).
+	targetCandidate := filter.NewCandidates(s.cluster.GetStores()).
```

Comment:
How about 
```suggestion
	targetCandidateWithTemp := filter.NewCandidates(s.cluster.GetStores()).
```

Reply:
I think it doesn't matter, `targetCandidateWithTemp` is too long.

Reply:
OK